dataland-frontend/src/components/resources/taxonomy/search/EuTaxoSearchResults.vue

```diff
@@ -131,4 +132,8 @@ export default {
 #search-result-taxonomy th {
   background: white;
 }
+
+.d-justify-content-end-inner > div {
+  justify-content: end;
+}
```

Comment:
Ich glaube wir hatten mal gesagt, dass wir custom styles scopen wollen.
Ich frage nochmal im Chat nach und schließe die Conversation ansonsten. Stephan müsste das noch wissen.

Reply:
Wenn ich die Klasse in ein <style scoped> packe, dann klappt das ganze aber nicht mehr. Ich glaube das liegt daran, dass die Klasse nicht in der Komponente, sondern in der PrimeVue Column-Componente genutzt wird

Reply:
Wie im Chat besprochen über ID-Zuordnung von anderen Komponenten fern halten.

Reply:
Done